packages/charting/src/components/DonutChart/Arc/Arc.types.ts

```diff
@@ -0,0 +1,57 @@
+import { IStyle } from 'office-ui-fabric-react/lib/Styling';
+import { IDataPoint } from '../DonutChart.types';
+
+export interface IArcProps {
+  /**
+   * Data to render in the chart.
```

Comment:
This comment is not descriptive, especially since you have the same 'data' in other interfaces with the same label but different types.